Dockerfile

```diff
@@ -8,5 +8,4 @@ RUN pip3 install pipenv
 RUN pipenv requirements > requirements.txt
 RUN pip3 install -r requirements.txt --target "${LAMBDA_TASK_ROOT}"
 
-# Default handler. See README for how to override to a different handler.
-CMD [ "lambdas.my_function.lambda_handler" ]
\ No newline at end of file
+CMD [ "apt.lambda_handler.handler" ]
```

Comment:
Hmm, should the CMD match the name of the function in `apt.lambda_handler`? 🤔

Reply:
Good catch!  Thanks.  

I had wanted to use `handler()` for the  actual function name, given it was part of `lambda_handler`... but I think I do prefer `apt.lambda_handler.lambda_handler` even it seems a little repetetive; could always add to `__init__` if desired.

Updated the Dockerfile!